packages/jaeger-ui/src/components/App/TopNav.tsx

```diff
@@ -96,9 +96,14 @@ function getItem(item: ConfigMenuItem) {
 function CustomNavDropdown({ label, items }: ConfigMenuGroup) {
   const menuItems = <Menu>{items.map(getItem)}</Menu>;
   return (
-    <Dropdown overlay={menuItems} placement="bottomRight">
-      <a>
-        {label} <DownOutlined />
+    <Dropdown
+      trigger={['click']}
+      overlay={menuItems}
+      overlayStyle={{ paddingRight: '20px' }}
+      placement="bottomRight"
+    >
+      <a style={{ color: 'white' }}>
```

Comment:
To change the color of `About Jaeger` from default green to white

Reply:
I don't think it's a good idea to override styles inline, it makes it impossible to provide skinning capability in the future (not that we need fully customizable skinning, but at least light mode / dark mode is a common feature these days).